components/Link/Link.tsx

```diff
@@ -0,0 +1,19 @@
+import { default as NextLink } from "next/link";
+import React from "react";
+import ArrowRightIcon from "public/icons/arrow-right.svg";
+import { Arrow, LinkSC } from "./styles";
+import { Typography, VariantsTypography } from "../../ui/typography";
+import { ILink } from "./types";
+
+export const Link = ({ href, text, variant }: ILink) => {
```

Comment:
not urgent for now, but you can specify ILink and add the ILinkProps to understand which type it is